src/core/dfn-finder.js

```diff
@@ -40,21 +40,57 @@ export function findDfn(defn, name, { parent = "" } = {}) {
  * @param {string} parent
  * @param {string} name
  */
-function getAlternativeNames(type, parent, name) {
+function getAlternativeNames(idlAst, parent, name) {
+  const { type } = idlAst;
   const asQualifiedName = `${parent}.${name}`;
   switch (type) {
     case "constructor":
     case "operation": {
-      // Allow linking to both "method()" and "method" name.
+      // Allow linking to "method()", method(arg) and "method" name.
       const asMethodName = `${name}()`;
       const asFullyQualifiedName = `${asQualifiedName}()`;
-      return [asFullyQualifiedName, asQualifiedName, asMethodName, name];
+      const asMethodWithArgs = deriveMethodWithArgs(name, idlAst.arguments);
+      return {
+        local: [asQualifiedName, asFullyQualifiedName, name],
+        exportable: [asMethodName, ...asMethodWithArgs],
+      };
     }
     case "attribute":
-      return [asQualifiedName, name];
+      return {
+        local: [asQualifiedName],
+        exportable: [name],
+      };
   }
 }
 
+function deriveMethodWithArgs(operationName, argsAst) {
```

Comment:
I think a comment to explain what this function does might be better. It's not clear from function name

Reply:
yeah, you are right ... will both rename and document.